plugin/spanreader/es/tagscontroller/tags_controller.go

```diff
@@ -328,6 +328,12 @@ func (r *tagsController) parseGetTagsValuesResponseBody(
 			for _, v := range aggregation["buckets"].([]any) {
 				bucket := v.(map[string]any)
 				value := bucket["key"]
+
+				// handle boolean fields
+				if kas, ok := bucket["key_as_string"]; ok {
```

Comment:
Will it show true or false for boolean params or 1 and 0?
Because I think we need to change this as well